### PR TITLE
Add documentation for system table optimizer_rule_stats

### PR DESCRIPTION
**Pull request by phd3**
cc @findepi

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 1]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
 and related metadata. This includes information such as the create time,
 idle time, initialization parameters, and accessed catalogs.
 
+``runtime.optimizer_rule_stats``
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:87]: Maybe move this section up so parameters are alphabetical for runtime.*

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 2]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The optimizer_rule_stats table contains the statistics for optimizer
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:90]:
```
``optimizer_rule_stats``
```

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 3]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
...
+rule invocations during query planning phase. The statistics are aggregated
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:91]: during the query

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 4]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
...
+over all queries since the server start-up, and keep getting updated.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:92]: Remove redundant section ", and keep getting updated"

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 5]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
...
+From this table you can find out invocation frequency, failure rates and
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:93]: The table contains information about ...

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 6]
```diff
@@ -84,6 +84,17 @@ The transactions table contains the list of currently open transactions
...
+performance for optimizer rules. For example, one can look at the
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/system.rst:94]: For example, you can look

**Comment by phd3** [comment]
@mosabua thanks for the review, fixed.

**Pull request merged**